tests/test_utils.py

```diff
@@ -382,20 +383,19 @@ def _call(*args, **kwargs):
         self.assertIn('--no-binary=:all:', pipargs)
 
     def test_echo_short_log_if_deployed(self):
-        log_file = Mock(delete=None)
         last_logs = ["last log line"]
 
         deployed = True
-        for verbose in [True, False]:
-            utils.echo_short_log_if_deployed(
-                deployed, last_logs, log_file, verbose)
-            self.assertEqual(None, log_file.delete)
+        for verbose, expected in zip([True, False], ["", "last log line\n"]):
```

Comment:
:lipstick: For static values, `zip` feels overkill.

```suggestion
        for verbose, expected in ((True, ""), (False, "last log line\n")):
```

Reply:
good point!